### DISCUSSION TITLE
Issue Triaging > Issue Board Process

### DISCUSSION THREAD

**Discussion by mefellows** [pact-foundation/roadmap#16] [Ideas]
TBC

**Comment by mefellows** [comment]
Example triage board: https://github.com/orgs/TheOdinProject/projects/12/views/1

_(via @JP-Ellis )_

**Reply by mefellows** [reply]
As discussed in the maintainer meet today, one idea was to have "open hours" where one or more of us get together and anyone is welcome to join in on the triage.

**Reply by JP-Ellis** [reply]
In addition to the labelling/prioritisation in the triage, this also provides a good opportunity to [move issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) between repos. Ideally, we don't want to force contributors to double their work just because they didn't find the correct repo in the first place.

**Reply by mefellows** [reply]
That's a really good point.
